modules/notes/controllers/getNotes.js

```diff
@@ -0,0 +1,20 @@
+import { getNotes } from "../../../db/providers/notesProvider.js";
+import logger from "../../../utils/logger.js";
+
+export const getListNotes = async (req, res) => {
+  const filter = {};
+  if (req.query.title) {
+    filter.title = req.query.title;
+  }
+
+  if (req.query.description) {
```

Comment:
Define title and description in a separate variables!
`const {...} = req.query;`